src/core/protocol-authorization.ts

```diff
@@ -198,21 +206,25 @@ export class ProtocolAuthorization {
       tenant,
       incomingMessage,
       ruleSet,
-      [], // ancestor chain is not relevant to subscriptions
+      [], // record chain is not relevant to subscriptions
```

Comment:
NIT while this comment is being updated
```suggestion
      [], // record chain is not relevant to queries or subscriptions
```

Reply:
Done.